docs/relay/relay.md

```diff
@@ -64,96 +83,70 @@ if (!AuthenticationService.Instance.IsSignedIn)
 }
 ```
 
-Here's the complete code that allows working with the [Relay server](https://docs.unity.com/relay/en/manual/relay-servers).
+## How do I join an existing allocation?
 
-```csharp
-/// <summary>
-/// RelayHostData represents the necessary information
-/// for a Host to host a game on a Relay
-/// </summary>
-public struct RelayHostData
-{
-    public string JoinCode;
-    public string IPv4Address;
-    public ushort Port;
-    public Guid AllocationID;
-    public byte[] AllocationIDBytes;
-    public byte[] ConnectionData;
-    public byte[] Key;
-}
+The host of your game created a Relay allocation, and your client has received its [join code](https://docs.unity.com/relay/en/manual/join-codes). You now need to [request all the allocation parameters](https://docs.unity.com/relay/manual/connection-flow#4) from the join code to join the game. To do that, call the `JoinAllocationAsync` method with your join code like so:
 
-/// <summary>
-/// HostGame allocates a Relay server and returns needed data to host the game
-/// </summary>
-/// <param name="maxConn">The maximum number of peer connections the host will allow</param>
-/// <returns>A Task returning the needed hosting data</returns>
-public static async Task<RelayHostData> HostGame(int maxConn)
-{
-    //Initialize the Unity Services engine
-    await UnityServices.InitializeAsync();
-    //Always autheticate your users beforehand
-    if (!AuthenticationService.Instance.IsSignedIn)
-    {
-        //If not already logged, log the user in
-        await AuthenticationService.Instance.SignInAnonymouslyAsync();
-    }
+```csharp
+//Ask Unity Services to join a Relay allocation based on our join code
+JoinAllocation allocation = await Unity.Services.Relay.RelayService.Instance.JoinAllocationAsync(joinCode);
+```
 
-    //Ask Unity Services to allocate a Relay server
-    Allocation allocation = await Unity.Services.Relay.RelayService.Instance.CreateAllocationAsync(maxConn);
+## How do I pass allocation data to my transport component?
 
-    //Populate the hosting data
-    RelayHostData data = new RelayHostData
-    {
-        // WARNING allocation.RelayServer is deprecated
-        IPv4Address = allocation.RelayServer.IpV4
-        Port = (ushort) allocation.RelayServer.Port,
+Now that you have an allocation (either by joining or hosting), you need to make all traffic coming from Netcode go through the Relay. To achieve that, you need to pass allocation parameters to your transport. For now, only the `UnityTransport` supports the Relay protocol. To pass allocation parameters to it, you first need to retrieve it from your `NetworkManager` like so:
```

Comment:
"Now that you have an allocation" - What have we allocated here? The relay server to an IP address?

Reply:
There is a link to the manual above. it is very short, but it mentions 

> An allocation is a request to the [Allocations service](https://docs.unity.com/ugs/en-us/manual/relay/manual/allocations-service) from the host player to reserve space on a [Relay server](https://docs.unity.com/ugs/en-us/manual/relay/manual/relay-servers) 

I understand it as "the server reserves space (and bandwidth?) to remember your IP and the ones of the other clients who connected so as to reroute all traffic". To make sure though, I would ask someone from UGS ;-)